src/Math-Complex/Number.extension.st

```diff
@@ -15,7 +15,12 @@ Number >> asComplex [
 
 { #category : #'*Math-Complex' }
 Number >> complexConjugate [
-	^ self.
+	"The complex conjugate of a complex number (a + bi) is another complex number (a - bi).
```

Comment:
I wouldn't have added the comment here and instead used tests to document this (Tests as Documentation). I would only comment on a design decision I made, the why, if you will.